src/libaudqt/info-widget.cc

```diff
@@ -55,6 +59,11 @@ static const TupleFieldMap tuple_field_map[] = {
     {N_("Codec"), Tuple::Codec, false},
     {N_("Quality"), Tuple::Quality, false},
     {N_("Bitrate"), Tuple::Bitrate, false},
+
+    {nullptr, Tuple::Invalid, false},
+    {N_("Folder"), Tuple::Path, false},
+    {N_("Filename"), Tuple::Basename, false},
+    {N_("Audio File URI"), Tuple::AudioFile, false},
```

Comment:
AudioFile is an internal field used for cuesheet support.  It's not meant to be displayed to the user.